cves/CVE-2012-0858.yml

```diff
@@ -64,7 +64,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: A file in ffmpeg allowed a denial of service and execution of arbitrary code because of an invalid free
```

Comment:
Definitely need more information here. Take a look at the "great work" tag for some examples of the amount of detail we're looking for. What part of the library would do allow this? What does an invalid free mean here? What would an attacker be able to do, potentially, to take advantage of this? How would an invalid free lead to arbitrary code execution?